client/my-sites/customer-home/cards/launchpad/pre-launch.tsx

```diff
@@ -16,26 +13,18 @@ const LaunchpadPreLaunch = ( props: LaunchpadPreLaunchProps ): JSX.Element => {
 	const siteId = useSelector( getSelectedSiteId ) || 0;
 	const site = useSelector( ( state: AppState ) => getSite( state, siteId ) );
 	const checklistSlug = site?.options?.site_intent ?? '';
-	const { data: allDomains = [] } = useGetDomainsQuery( site?.ID ?? null, {
-		retry: false,
-	} );
+	const dispatch = useDispatch();
 
-	const layout = useHomeLayoutQuery( siteId || null );
-	const { isOpen, setModalIsOpen, handleSiteLaunched } = useCelebrateLaunchModal( siteId, layout );
+	const handleSiteLaunched = () => {
+		dispatch( launchSiteOrRedirectToLaunchSignupFlow( siteId, 'home' ) );
```

Comment:
Same thing here: We're reusing CelebrateLaunchModal in the top Home component and reusing the `launchSiteOrRedirectToLaunchSignupFlow` action taking advantage that the logic for launching or redirecting will be implemented only in one place inside of it.